tokenomics.md

```diff
@@ -27,23 +27,21 @@ Token allocation is benchmarked to similar successful projects. Typically, such
     2. Aggregators.
```

Comment:
inflationary block rewards should go to aggregators already. So I would remove that